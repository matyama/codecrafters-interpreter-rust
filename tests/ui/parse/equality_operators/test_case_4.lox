(70 != 92) == ((-28 + 51) >= (34 * 38))
