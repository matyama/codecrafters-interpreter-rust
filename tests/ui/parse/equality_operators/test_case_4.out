(== (group (!= 70.0 92.0)) (group (>= (group (+ (- 28.0) 51.0)) (group (* 34.0 38.0)))))
